painttiming.bs

```diff
@@ -177,8 +182,9 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
-This allows developers to detect support for paint timing.
+NOTE: A user agent implementing {{PerformancePaintTiming}} must include <code>"paint"</code> in the [=frozen array of entry types=],
+if its [=relevant global object=] is associated with a [=top-level browsing context=]. It may also include <code>"paint"</code> for any [=nested browsing context=].
+This allows developers to detect support for paint timing for a particular [=document=], and allows user-agents to configure availability of paint-timing for iframes.
 
```

Comment:
@npm1 @yoavweiss 
Made another attempt, trying to be as precise as I can :) 
Thanks for your patience!